### PR TITLE
Adds first LDK blog post

### PR DESCRIPTION
**Pull request by ConorOkus**
Preview [here](https://deploy-preview-126--lightningdevkit.netlify.app/blog/ldk-an-sdk-for-the-lightning-network/)

### CODE REVIEW

File: docs/_blog/ldk-an-sdk-for-the-lightning-network.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+`Taproot`  
+Taproot! People love it. We love it. It’s been several months since full activation, and LDK will start to take advantage of everything it offers. This includes improved on-chain privacy by making current funding channel outputs (2-of-2 multisig) indistinguishable from single sig outputs, and making channel opening and closing transactions look more like regular non-lightning transactions. In addition, we’re looking to support multi-sig channel capabilities as they allow for greater security, redundancy, and flexibility.
+
+`Language Bindings`  
+We’re making existing language integrations feel more native. We currently support Rust, C/C++, Swift, Java, & Kotlin. It’s now in the beta stage, but JavaScript/TypeScript/WASM is also supported, and we are prioritizing bug fixing for early adopters. In the future, expect support for C#, Python, Flutter/Dart, and possibly others.
```
Comment (dunxen) [inline docs/_blog/ldk-an-sdk-for-the-lightning-network.md:72]: Tiny nit: WASM should be Wasm

File: docs/_blog/ldk-an-sdk-for-the-lightning-network.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+LDK supports applications that require running many nodes (such as server/custodial apps), letting users run nodes on mobile devices, integrating with existing infrastructure, broad customization options, and resource-constrained embedded systems such as point of sale, IoT, and so on and so on, etc.  
+
+[Blue Wallet](https://bluewallet.io) was LDK’s patient zero and ideal first adopter since it is both non-custodial and mobile. Their team maintains [rn-ldk](https://github.com/BlueWallet/rn-ldk) and runs it on both Android and iOS.The wallet is likely to ship with LDK soon but not before some critical infrastructure is in place. While non-custodial mobile wallets are LDK’s primary use case, that doesn’t make it limited to them.
+
+[CashApp](https://cash.app), one of the world’s most used payment apps, chose LDK for its Lightning implementation because of its customizability. While custodial server-based solutions are not the primary use case for LDK, it shows how adaptable the library is. Ryan Loomba recently explained some of the trade-offs their team had to think about when comparing LDK to other solutions [at bitcoin++](https://www.youtube.com/watch?v=5YmfEgh-LC8&t=2442s).
```
Comment (jkczyz) [inline docs/_blog/ldk-an-sdk-for-the-lightning-network.md:52]: Should we link the Spiral talk somewhere? Or is there a concern with sound quality?

Comment (jkczyz) [reply docs/_blog/ldk-an-sdk-for-the-lightning-network.md:52]: Or maybe not the right blob post for it?

Comment (ConorOkus) [reply docs/_blog/ldk-an-sdk-for-the-lightning-network.md:52]: Wait which Spiral talk?

Comment (jkczyz) [reply docs/_blog/ldk-an-sdk-for-the-lightning-network.md:52]: Ah, I meant the LDK talk by Sprial (Arik and me).

Comment (ConorOkus) [reply docs/_blog/ldk-an-sdk-for-the-lightning-network.md:52]: Oh okay sure, will add that in 💯

**Pull request merged**